gato/training/arguments.py

```diff
@@ -0,0 +1,271 @@
+from dataclasses import dataclass, field
+from typing import List, Literal, Optional
+
+@dataclass
+class TrainingArgs:
```

Comment:
I know this docstring is close to duplication of the arguments themselves. I guess I'm hoping the docstring can be fleshed out with more context and links, and maybe even get rendered as markdown in some documentation at some point.

But I'm not strongly committed to that. So if you have feelings about this, I'm all ears.

Reply:
I would consider this as duplication. I would much prefer the metadata.help for providing any documentation/detail. That makes people think how to summarise what they have, plus anyone committing code would easily see that as a pattern, and not the docstring necessarily.

Can we just use "help" parameter as you've used for a couple and put documentation for each parameter/arg there?

Reply:
Yup. This cleanup looks way better.

I was looking for a place to put annotations/notes/comments that didn't belong in the CLI "help".

For example, these comments regarding the interactions between `mu` and `M`.

```
    # Mu-law companding. See section 2.1 Tokenization of the GATO paper.
    # Values described in the GATO paper (100, 256) cause a rapid ~convergence to values between -0.7 and 0.7.
    mu: int = field(default=100, metadata={'help': 'Higher values cause more rapid sloping.'})
    M: int = field(default=256, metadata={'help': 'Higher values cause smaller upper and lower bounds.'})
```

But I think the comment next to the definition works better than having so much duplication in the docstring.

Reply:
Yep inline comments or just above the definition makes more sense. :)